renderer/index.js

```diff
@@ -96,6 +96,14 @@ function init () {
       }
     } else if (e.which === 32) { /* spacebar pauses or plays the video */
       dispatch('playPause')
+    } else if ((e.ctrlKey || e.metaKey) && e.altKey && e.which === 39) {
```

Comment:
I would make these menu items---that way users will be able to discover what the shortcuts are. See `menu.js`